VERSION

```diff
@@ -1,2 +1,2 @@
 PKG_VERSION="2.5.0"
-
+JMX_METRICS_JAR_VERSION="1.6.0"
```

Comment:
This is an internal implementation detail. I think you're right that we should add a `BUILD_CONFIG` file or something. Unless we can extract this info from the `opentelemetry-java-contrib` submodule sources…

Reply:
I'll see if I can find a good way to dig it out of source, otherwise will refactor to a BUILD_CONFIG file

Reply:
This version string is meaningless as it stands, you could make it ANY valid number.  This should not be a blocking comment.

Reply:
Specifically

- The version string is NOT used to generate properties files or any runtime version number (they're still hardcoded)
- The version string is erased on the JAR when we bundle it in ops-agent
- The string just needs to be three valid numbers right now.
- We plan to migrate this build out of the ops-agent buid with an explicit verson choice that's not tied to git submodule.   While there's still an issue with the nebula plugin in gradlew, we're stuck with this kind of hack right now.

Reply:
No luck on digging out of source, my understanding is that the main point of the nebula plugin is that versioning is all stored in git tags, nothing in source.

Reply:
Sam - I see you moved it into BUILD_CONFIG.   Unfortuantely that version number isn't actually a valid config parameter.

Can you add a comment to it that it's ONLY used for build wiring but has no impact on the version selected/used?

Reply:
@jsuereth the way it's used in the Powershell scripting for the Windows build might not play very nice with a comment without some more significant changes, but I can give it a try.

@igorpeshansky we're using this workaround to specify version rather than allowing Nebula plugin to define version because the way CI builds the images doesn't download git config/history for submodules.